### DISCUSSION TITLE
Feature Suggestion: Mount Google Drive / Teamdrive via rclone

### DISCUSSION THREAD

**Discussion by Dud3z** [retronas/discussions#93] [Ideas]
Hi, i have a cloud only setup at home. Everything is in a Google Team Drive. All my Games, All my Mister Stuff.
rn i either copy/backup everything by hand or on the mister directly.

it would be VERY cool if i could just mount the retronas to my mister and the retronas is acting like a kind of proxy.

**Comment by sairuk** [comment]
Interestingly, I thought we had an open discussion about this but I couldn't find it.

I believe @danmons was looking into the feasibility of rclone as a future idea, it is currently listed in [IDEAS.md](https://github.com/danmons/retronas/blob/main/IDEAS.md) under cloud drives
